SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/Tree/TreeFactory.swift

```diff
@@ -9,14 +9,66 @@
 import Foundation
 import UIKit
 
+let nouns:[String] = [
```

Comment:
I know this array is ugly. I just stuck it in for now to test the voice on actual words.